content/en/tracing/trace_collection/custom_instrumentation/nodejs/otel.md

```diff
@@ -66,22 +53,87 @@ The following OpenTelemetry features implemented in the Datadog library as noted
     provider.register()
     ```
 
-6. Run your application.
+6. Import the OpenTelemetry API and create an OpenTelemetry tracer instance:
+
+    ```js
+    const ot = require('@opentelemetry/api')
+    const otelTracer = ot.trace.getTracer(
+      'my-service'
+    )
+    ```
+
+7. Run your application.
+
+Datadog combines these OpenTelemetry spans with other Datadog APM spans into a single trace of your application. It also supports [integration instrumentation][2] and [OpenTelemetry automatic instrumentation][3].
+
+## Adding span tags
+
+Add custom attributes to your spans to provide additional context:
+
+{{< highlight js "hl_lines=6" >}}
+function processData(i, param1, param2) {
+  return tracer.startActiveSpan(`processData:${i}`, (span) => {
+    const result = someOperation(param1, param2);
+
+    // Add an attribute to the span
+    span.setAttribute('app.processedData', result.toString());
+    
+    span.end();
+    return result;
+    });
+}
+{{< /highlight >}}
+
+## Creating spans
+
+To create a new span and properly close it, use the `startActiveSpan` method:
+
+{{< highlight js "hl_lines=3 9" >}}
+function performTask(iterations, param1, param2) {
+  // Create a span. A span must be closed.
+  return tracer.startActiveSpan('performTask', (span) => {
+    const results = [];
+    for (let i = 0; i < iterations; i++) {
+      results.push(processData(i, param1, param2));
+    }
+    // Be sure to end the span!
+    span.end();
+    return results;
+  });
+}
+{{< /highlight >}}
+
```

Comment:
This page has all the code examples for how to start a span:

https://datadoghq.dev/dd-trace-js/interfaces/export_.opentelemetry.Tracer.html

One important thing to note and mention is that the nodejs tracer doesn't currently support the Otel Context API, so user's should be advised against creating spans from otel contexts.